src/applications/hca/containers/IntroductionPageGated.jsx

```diff
@@ -114,7 +116,13 @@ const LoggedOutContent = connect(
       downtime={route.formConfig.downtime}
     />
     <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
-      <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018" />
+      {environment.isProduction() ? (
+        <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018" />
+      ) : (
+        <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018">
+          <HealthcareModalContent resBurden={30} ombNumber="2900-0091" />
```

Comment:
link

Reply:
"link"?

Reply:
yeah I needed to link to this in another PR and AFAIK there's not another easy way .